_episodes/08-development-setup.md

```diff
@@ -53,7 +53,10 @@ Let’s get started.
 
 The ESMValTool source code is available on a public GitHub repository:
 [https://github.com/ESMValGroup/ESMValTool](https://github.com/ESMValGroup/ESMValTool).
-To obtain the code, the easiest way is to clone the repository:
+To obtain the code, there are two options:
+
+1. download the code as a ZIP file from the repository.
```

Comment:
Perhaps there should be some details on what happens after the zip file is downloaded or at what point it becomes the same as in the GitHub installation steps?

Reply:
The description is revised.